src/java/com/puppetlabs/ssl_utils/SSLUtils.java

```diff
@@ -288,12 +289,11 @@ public static X509Certificate signCertificate(String issuerDn,
     public static X509CRL generateCRL(X500Principal issuer,
                                       PrivateKey issuerPrivateKey,
                                       PublicKey issuerPublicKey,
+                                      Date thisUpdate,
```

Comment:
Can we maybe keep calling this `issueDate`? That's a little clearer to me than this name.

Reply:
I went back and forth about this. I agree with you that `issueDate` is clearer, but `thisUpdate`, like `nextUpdate`, is what the actual CRL attribute is called. Curious what others think.

Reply:
Oh okay, I didn't know that. Then yeah, I think I support your choice.